salt/client/mixins.py

```diff
@@ -341,40 +343,3 @@ def print_async_event(self, suffix, event):
             event.pop('_stamp')  # remove the timestamp before printing
```

Comment:
we are thinking about keeping the _stamp in the printed data, and not doing the pop with @cachedout.

What's your point @jacksontj ?